accumulo/src/main/java/org/vertexium/accumulo/AccumuloElement.java

```diff
@@ -153,6 +153,7 @@ protected void setVisibility(Visibility visibility) {
         );
 
         if (mutation.getNewElementVisibility() != null) {
+            getGraph().flush();
```

Comment:
can we add a comment for why we added this graph.flush here to remind us of the reason?

Reply:
@sfeng88 Done with https://github.com/visallo/vertexium/pull/183/commits/3934f54bb4dd96a8300bb5a171a9d37e2cb9d9a4